homeassistant/components/nws/weather.py

```diff
@@ -84,17 +84,15 @@ async def async_setup_entry(
     entity_registry = er.async_get(hass)
     nws_data: NWSData = hass.data[DOMAIN][entry.entry_id]
 
-    entities = [NWSWeather(entry.data, nws_data, DAYNIGHT)]
-
-    # Add hourly entity to legacy config entries
-    if entity_registry.async_get_entity_id(
+    # Remove hourly entity from legacy config entries
+    if entity_id := entity_registry.async_get_entity_id(
```

Comment:
At some point we can assume that users will no longer have this entity.  Should we aim to remove this code at some point, and add a comment to this effect?  Or are we committing to keeping this around?

Reply:
6 releases is the general deprecation time so I guess that's relevant here too for cleaning this up eventually.